audio/FAudio/files/faudio.pc

```diff
@@ -0,0 +1,12 @@
+prefix=@PREFIX@
+exec_prefix=${prefix}
```

Comment:
when this file is copied in, where is this ${prefix} ever changed to "${prefix}"?

Reply:
likewise with the other references. I think they all have to be @prefix@ and then reinplaced after.

Reply:
Doing this is standard procedure within pkg-config files.

The initial `prefix` Sets the variable ${prefix} that the other sections use.

Reply:
Oh, you're right. That's interesting, I never looked closely at them I guess. I was thinking for that to work they'd have to be ```$(PREFIX)``` type references, but obviously not.